sympy/calculus/tests/test_singularities.py

```diff
@@ -43,6 +43,7 @@ def test_is_increasing():
     assert is_increasing(1)
 
     assert is_increasing(4*x**3 - 6*x**2 - 72*x + 30, Interval(-2, 3)) is False
+    assert is_increasing(x**5 -5*x +7,Union(Interval(-oo, -1), Interval(1, oo))) is True
```

Comment:
this is already passing in master; I wouldn't add this unless it improves coverage somewher